common/post-processing-worker-filter.h

```diff
@@ -50,7 +59,11 @@ class post_processing_worker_filter : public post_processing_filter
                     continue;
                 if( !f )
                     continue;
-                worker_body( f.as< rs2::frameset >() );
+                // TODO Noha: why is the subdevice model needed?
```

Comment:
Noha, I still don't understand what the dependency on the subdevice_model here is, and why the worker_filter needs to know about it...